litert/vendors/qualcomm/core/wrappers/tensor_wrapper.cc

```diff
@@ -25,6 +25,23 @@
 
 namespace qnn {
 
+bool IsNBitQuant(const QuantizeParamsWrapperVariant& quantize_params,
```

Comment:
Nit: put it in anonymous namespace.

```suggestion
bool IsNBitwiseQuant(const QuantizeParamsWrapperVariant& quantize_params,
```